src/components/elements/CommentaryBox/CommentaryBox.tsx

```diff
@@ -35,26 +97,76 @@ const CommentaryBox = (props: CommentaryBoxProps) => {
           className="max-h-72 !min-h-0 resize-none border-none !p-0 text-xs"
           containerClassName="w-full"
           rows={1}
+          onChange={e => handleCommentChange(e)}
+          value={comment}
         />
         <label htmlFor="input-files" className="cursor-pointer">
           <input
             type="file"
             id="input-files"
             className="absolute z-[-1] h-[0.1px] w-[0.1px] overflow-hidden opacity-0"
+            onChange={e => {
+              if (e.target.files) {
+                handleFileChange(e);
+              }
+            }}
           />
           <Icon name={IconNames.PAPER_CLIP} className="h-4 w-4" />
         </label>
         <When condition={buttonSendOnBox}>
           <Button variant="text" iconProps={{ name: IconNames.SEND, className: "h-4 w-4 text-darkCustom" }} />
         </When>
       </div>
+      <div className="flex flex-1 justify-between gap-4">
+        <div className="flex flex-wrap gap-2">
+          <When condition={files.length > 0}>
+            {...files.map((file: any) => (
```

Comment:
Is this `...` spread doing anything useful? I think it's superfluous.

Reply:
done